### PR TITLE
Add /location/:iso/time_series_values endpoint

### PR DESCRIPTION
**Pull request by ticklemynausea**
Url is `/api/v1/location/:iso/time_series_values`

Response sample: https://gist.githubusercontent.com/ticklemynausea/007d0767e1a10676f05fa1773edfe02c/raw/164bcb8f49e30dd9bf177e8b0137a2eae269662b/-

It supports an option `secenario`query parameter, which accepts a comma-separated list of scenario ids, which will be used to filter TimeSeriesValues.

### CODE REVIEW

File: app/controllers/api/v1/location_time_series_values_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        render json: time_series_values,
+               each_serializer: Api::V1::LocationTimeSeriesValuesSerializer
+      end
+
+      private
+
+      def set_location
+        @location = Location.
+          includes(:time_series_values).
+          find_by!(iso_code: params[:location_id])
```
Comment (Bluesmile82) [inline app/controllers/api/v1/location_time_series_values_controller.rb]: This should be: find(params[:location_id]) as we are using the location_id, not the iso code

File: app/serializers/api/v1/location_time_series_values_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      def indicator_id
+        object.first
+      end
+
+      def values
+        object.second.map do |tsv|
+          {
+            year: tsv.year,
+            value: tsv.value,
+            scenario_id: tsv.scenario_id
```
Comment (Bluesmile82) [inline app/serializers/api/v1/location_time_series_values_serializer.rb]: Could we skip this scenario_id if we filter by scenario?

Comment (ticklemynausea) [reply app/serializers/api/v1/location_time_series_values_serializer.rb]: done!

File: app/controllers/api/v1/location_time_series_values_controller.rb (hunk 1 above)
Comment (ticklemynausea) [reply app/controllers/api/v1/location_time_series_values_controller.rb]: fixed

**Comment by ticklemynausea** [comment]
@Bluesmile82  the branch has been updated with the changes you requested

**Pull request merged**